executor/generic.go

```diff
@@ -204,6 +204,10 @@ func DefaultEnv(namespace string) EnvVarConverter {
 	defaults.SetString(cfg.AwsSecretAccessKeyEnvName, cfg.Config.GlobalSecretAccessKey)
 	defaults.SetString("HOSTNAME", namespace)
 
+	if cfg.Config.ResticOptions != "" {
+		defaults.SetString(cfg.ResticOptionsEnvName, cfg.Config.ResticOptions)
+	}
```

Comment:
Is there a harm if the environment variable is defined but empty? We could skip the conditional